src/importers/onepassword1PifImporter.ts

```diff
@@ -120,6 +121,15 @@ export class OnePassword1PifImporter extends BaseImporter implements Importer {
                     }
                 });
             }
+            if (item.secureContents.passwordHistory != null) {
+                cipher.passwordHistory = cipher.passwordHistory || [];
+                item.secureContents.passwordHistory.forEach((entry: any) => {
+                    const phv = new PasswordHistoryView();
+                    phv.password = entry.value;
+                    phv.lastUsedDate = new Date(entry.time * 1000);
```

Comment:
Is there utility code / a smarter way to create the correct timestamp here?

Reply:
No, this should be fine.